Base/QTGUI/qSlicerExtensionsLocalWidget.cxx

```diff
@@ -701,15 +710,29 @@ QListWidgetItem* qSlicerExtensionsLocalWidgetPrivate::updateExtensionItem(const
     qWarning() << Q_FUNC_INFO << " failed: missing implementation for serverAPI" << serverAPI;
     }
 
+  QString installedRevision = metadata["installed"].toBool() ? metadata["revision"].toString() : QString();
+  QString onServerRevision = metadataFromServer["revision"].toString();
+
+  // Generate change log URL - only if repository is hosted on github
+  QUrl scmUrl = QUrl(metadataFromServer["scmurl"].toString());
+  QString changeLogUrl;
+  if (scmUrl.host().toLower() == "github.com"
```

Comment:
Should be easy to support the couple of extensions on gitlab.com as well

Reply:
Good idea. It would be great if you could look it up how it can be done.

Reply:
The same convention is used for gitlab.

`scmUrl.setPath(QString("%1/compare/%2...%3").arg(scmUrl.path()).arg(installedRevision).arg(onServerRevision));`

For example:
https://gitlab.com/opendose/opendose3d/compare/2d08584afbb71d829d63a00e9f1c7c7898d3b8c5...2a9ebce68627f96da05dda628bada05c3ef827df
which can be used and gets redirected to:
https://gitlab.com/opendose/opendose3d/-/compare/2d08584afbb71d829d63a00e9f1c7c7898d3b8c5...2a9ebce68627f96da05dda628bada05c3ef827df